src/lib/learn-client/types.ts

```diff
@@ -79,7 +79,7 @@ export type CollectionLite = Pick<
 >
 
 export interface Collection {
-	id?: uuid // uuid
+	id: uuid // uuid
```

Comment:
Collection `id` is required for Tutorial progress to work. I think it's not "optional" from a `learn-client` perspective, as it's always available from the API.